configs/nifi.properties

```diff
@@ -147,9 +147,9 @@ nifi.web.jetty.threads=200
 # nifi.web.proxy.context.path=
 
 # security properties #
-nifi.sensitive.props.key=
+nifi.sensitive.props.key={{.Values.properties.sensitiveKey}}
 nifi.sensitive.props.key.protected=
-nifi.sensitive.props.algorithm=PBEWITHMD5AND256BITAES-CBC-OPENSSL
+nifi.sensitive.props.algorithm=NIFI_PBKDF2_AES_GCM_256
```

Comment:
If you're changing a default, would you please make it configurable so we can continue to use the old default if we already have a bunch of things protected with a key?